src/d4bl/query/fusion.py

```
@@ -0,0 +1,160 @@
+                f"{self.ollama_base_url}/api/generate",
+                json={
+                    "model": "mistral",
+                    "prompt": prompt,
+                    "stream": False,
+                    "options": {"temperature": 0.3},
+                },
+            ) as response:
+                if response.status != 200:
+                    raise RuntimeError(
+                        f"Ollama returned status {response.status}"
+                    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)